daal4py/sklearn/monkeypatch/dispatcher.py

```diff
@@ -81,7 +83,9 @@
     'lasso':         [[(linear_model_module, 'Lasso', Lasso_daal4py), None]],
     'svm':           [[(svm_module, 'SVC', SVC_daal4py), None]],
     'logistic':      [[(logistic_module, _patched_log_reg_path_func_name, daal_optimized_logistic_path), None]],
-    'knn_clsf':      [[(neighbors_module, 'KNeighborsClassifier', KNeighborsClassifier_daal4py), None]]
+    'knn_clsf':      [[(neighbors_module, 'KNeighborsClassifier', KNeighborsClassifier_daal4py), None]],
+    'nrst_nbrs':     [[(neighbors_module, 'NearestNeighbors', NearestNeighbors_daal4py), None]],
```

Comment:
These are the names that the user controls when choosing what he wants to patch. I would recommend giving full names.
```python
'nearest_neighbors':     [[(neighbors_module, 'NearestNeighbors', NearestNeighbors_daal4py), None]],
```

Reply:
Fixed